Src/Fido2/AttestationFormat/AndroidKey.cs

```diff
@@ -21,25 +23,34 @@ public static byte[] AttestationExtensionBytes(X509ExtensionCollection exts)
             }
             return null;
         }
+
         public static byte[] GetASN1ObjectAtIndex(byte[] attExtBytes, int index)
         {
             // https://developer.android.com/training/articles/security-key-attestation#certificate_schema
             // This function returns an entry from the KeyDescription at index
-            if (null == attExtBytes || 0 == attExtBytes.Length || attExtBytes.Length > UInt16.MaxValue) throw new Fido2VerificationException("Invalid attExtBytes signature value");
+            if (null == attExtBytes || 0 == attExtBytes.Length || attExtBytes.Length > ushort.MaxValue)
+                throw new Fido2VerificationException("Invalid attExtBytes signature value");
             var offset = 0;
             var derSequence = AuthDataHelper.GetSizedByteArray(attExtBytes, ref offset, 1);
+
             // expecting to start with 0x30 indicating SEQUENCE
-            if (null == derSequence || 0x30 != derSequence[0]) throw new Fido2VerificationException("attExtBytes signature not a valid DER sequence");
+            if (null == derSequence || 0x30 != derSequence[0])
+                throw new Fido2VerificationException("attExtBytes signature not a valid DER sequence");
+
             // next is length of all the items in the sequence
             var dataLen = AuthDataHelper.GetSizedByteArray(attExtBytes, ref offset, 1);
-            if (null == dataLen) throw new Fido2VerificationException("attExtBytes signature has invalid length");
+            if (null == dataLen)
+                throw new Fido2VerificationException("attExtBytes signature has invalid length");
+
             // if data is more than 127 bytes, the length is encoded in long form
+            // TODO : Why is longLen never used ?
```

Comment:
Because I forgot to bring over the sanity check added on that field I added in #100.